packages/sources/tradingeconomics/test/integration/adapter.test.ts

```diff
@@ -4,16 +4,28 @@ import * as process from 'process'
 import { server as startServer } from '../../src'
 import * as nock from 'nock'
 import * as http from 'http'
-import { mockResponseSuccess } from './fixtures'
+import { mockResponseSuccess, mockSubscribeRequest, mockUnsubscribeRequest } from './fixtures'
 import { AddressInfo } from 'net'
+import {
+  mockWebSocketProvider,
+  mockWebSocketServer,
+  MockWsServer,
+  mockWebSocketFlow,
+} from '@chainlink/ea-test-helpers'
+import { WebSocketClassProvider } from '@chainlink/ea-bootstrap/dist/lib/middleware/ws/recorder'
+import { DEFAULT_WS_API_ENDPOINT } from '../../src/config'
+
+const sleep = (ms: number) => {
+  return new Promise((resolve) => setTimeout(resolve, ms))
+}
 
 describe('execute', () => {
   const id = '1'
   let server: http.Server
   let req: SuperTest<Test>
 
   beforeAll(async () => {
-    process.env.CACHE_ENABLED = 'false'
+    // process.env.CACHE_ENABLED = 'false'
```

Comment:
Should this line be commented?

Reply:
@eddiejhong It's good that you brought this question. Because of websocket integration tests i had to enable cache. Also i don't see any reason why we should disable cache even for  regular integration tests

Reply:
Websockets work by topping up the cache with WS data - so it makes sense that they won't work if the cache is off.